sample/app/src/main/java/com/yoti/mobile/android/capture/face/demo/FaceCaptureActivity.kt

```diff
@@ -0,0 +1,298 @@
+        cameraConfiguration = CameraConfiguration(facing = if (captureConfigExtra.isBackCamera) BACK else FRONT)
+        configuration = FaceCaptureConfiguration(
+                faceCenter = captureConfigExtra.faceCenter,
+                imageQuality = captureConfigExtra.imageQuality,
+                requireValidAngle = captureConfigExtra.requireValidAngle,
+                requireEyesOpen = captureConfigExtra.requireEyesOpen,
+                requireBrightEnvironment = captureConfigExtra.requireBrightEnvironment,
+                requiredStableFrames = if (captureConfigExtra.requireStableFrames) 3 else 1,
+                provideLandmarks =  captureConfigExtra.provideLandmarks,
+                provideSmileScore = captureConfigExtra.provideSmileScore
+        )
+
+
+        with(binding) {
+            manualCaptureButton.setOnClickListener {
+                isFirstFrameValid = true
+            }
+        }
+    }
+
+    override fun onResume() {
+        super.onResume()
+        clearState()
+        startCamera()
+        binding.faceCaptureView.startAnalysing(configuration, ::onFaceCaptureResult)
+    }
+
+    override fun onPause() {
+        stopAnalyzing()
+        clearState()
+        super.onPause()
+    }
+
+    private fun startCamera() {
+        binding.faceCaptureView.startCamera(this, ::onCameraState, cameraConfiguration)
+    }
+
+    private fun stopAnalyzing() = with(binding) {
+        if (faceCaptureView.isAnalysing()) {
+            faceCaptureView.stopAnalysing()
+        }
+    }
+
+    private fun logState(@StringRes state: Int) {
+        binding.userFeedback.setText(state)
+    }
+
+    private fun clearState() {
+        binding.userFeedback.text = ""
+        if (isDebugMode) {
+            clearOverlay()
+        }
+    }
+
+    private fun onCameraState(state: CameraState) {
+        when (state) {
+            is CameraInitializationError -> logState(handleCameraInitError(state.cause))
+            is MissingPermissions -> logState(R.string.yoti_fcm_demo_camera_error_permissions)
+            else -> {
+            }
+        }
+    }
+
+    private fun handleCameraInitError(error: CameraError) = when (error) {
+        IllegalState -> R.string.yoti_fcm_demo_camera_error_illegal_state
+        UnableToResolveCamera -> R.string.yoti_fcm_demo_camera_error_unable_to_resolve
+        else -> R.string.yoti_fcm_demo_camera_error_unknown
+    }
+
+    private fun onFaceCaptureResult(result: FaceCaptureResult) {
+        when (val state = result.state) {
+            is InvalidFace -> handleInvalidFace(state.cause, result.originalImage)
+            is ValidFace -> handleValidFace(state, result.originalImage)
+        }
+    }
+
+    private fun handleValidFace(result: ValidFace, originalImage: ImageBuffer? = null) {
+        logState(R.string.yoti_fcm_demo_result_valid_face)
+
+        if (isFirstFrameValid) {
+            Glide.with(this@FaceCaptureActivity).load(result.croppedImage).into(binding.captureResult)
```

Comment:
Using Third party libraries like Glide in a sample app is confusing because it makes it look like they're required to use Glide to interact with our SDK which is not true at all. All we need here is BitmapFactory.decode and setBitmap